src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Specialized;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+
+namespace NzbDrone.Core.Notifications.Notifiarr
+{
+    public interface INotifiarrProxy
+    {
+        void SendNotification(StringDictionary message, NotifiarrSettings settings);
+        ValidationFailure Test(NotifiarrSettings settings);
+    }
+
+    public class NotifiarrProxy : INotifiarrProxy
+    {
+        private const string URL = "https://notfiarr.com";
+        private readonly IHttpClient _httpClient;
+        private readonly IConfigFileProvider _configFileProvider;
+        private readonly Logger _logger;
+
+        public NotifiarrProxy(IHttpClient httpClient, IConfigFileProvider configFileProvider, Logger logger)
+        {
+            _httpClient = httpClient;
+            _configFileProvider = configFileProvider;
+            _logger = logger;
+        }
+
+        public void SendNotification(StringDictionary message, NotifiarrSettings settings)
+        {
+            try
+            {
+                ProcessNotification(message, settings);
+            }
+            catch (NotifiarrException ex)
```

Comment:
This will result in every `HttpException` that is thrown by `ProcessNotification` getting logged twice. Does this need to capture the exception and throw it again with just the message or can it skip that?

Reply:
The double logging is actually something that's been annoying, but didn't know the route to correct. Thanks

I'll test, but this should be able to just be passthrough then a simple throw